### PR TITLE
REPLACE babel/runtime with core-js

### PR DESCRIPTION
**Pull request by ndelangen**
Issue: https://github.com/storybooks/storybook/issues/4088

## What I did
I removed the alias for babel in the preview webpack config

## How to test
everything should just work
run the fixtures / examples
check on netlify / teamcity

### CODE REVIEW

**Review by Hypnosphi — CHANGES_REQUESTED** [review]
Please either add `@babel/runtime` as a peer dependency to all packages which have a `preview` part and add it to CLI generators, or just remove `transform-runtime` both from default babel config for user's storybook and from our own root config (which transpiles our sources)

File: .babelrc.js [hunk 1]
```diff
@@ -25,7 +25,10 @@ module.exports = {
   overrides: [
     {
       test: './examples/vue-kitchen-sink',
-      presets: [['@babel/preset-env', { shippedProposals: true }], 'babel-preset-vue'],
+      presets: [
+        ['@babel/preset-env', { shippedProposals: true, useBuiltins: 'usage' }],
```
Comment (Hypnosphi) [inline .babelrc.js]: Since you use the same settings as on top level, you can omit this preset in the override

Comment (ndelangen) [reply .babelrc.js]: Really? babel merges these?

Comment (Hypnosphi) [reply .babelrc.js]: yep https://babeljs.io/docs/en/options#merging

**Review by Hypnosphi — APPROVED** [review]

File: examples/angular-cli/angularshots.test.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+// Needed to prevent "Zone.js has detected that ZoneAwarePromise `(window|global).Promise` has been overwritten."
+import 'core-js/modules/es6.promise';
```
[the remark this answers is not in the document]
Comment (Hypnosphi) [reply examples/angular-cli/angularshots.test.js]: IDK =(

[the remark this answers is not in the document]
Comment (igor-dv) [reply examples/angular-cli/angularshots.test.js]: Maybe in `addons/storyshots/storyshots-core/src/frameworks/angular/loader.js` will help 🤔

[the remark this answers is not in the document]
Comment (igor-dv) [reply examples/angular-cli/angularshots.test.js]: Yeah, will check this

**Pull request merged**